app/views/solutions/show.html.haml

```diff
@@ -88,31 +90,33 @@
               %li What compromises have been made?
               %li Are there new concepts here that you could read more about to improve your understanding?
 
-  -if @comments.size > 0
-    .comments-section
-      .lo-container
-        .pure-g
-          .pure-u-1.pure-u-md-1-2
-            %h2 Community comments
-            .h2-subtitle See what others have said about this solution
-            -@comments.each do |reaction|
-              .comment.pure-g
-                .pure-u-1.pure-u-md-2-24
-                  -if reaction.emotion == "like"
-                    =icon 'thumbs-up'
-                  -elsif reaction.emotion == "love"
-                    =icon 'heart'
-                  -elsif reaction.emotion == "genius"
-                    =graphical_image "genius-turquoise.png"
-                  -else reaction.emotion == "legacy"
-                    =icon 'comment'
 
-                .pure-u-1.pure-u-md-22-24.content
-                  .when #{time_ago_in_words(reaction.created_at)} ago
-                  .who
-                    %strong #{reaction.user.handle}
-                    says
-                  .text= simple_format reaction.comment
+  -#
+    -if @comments.size > 0
+      .comments-section
+        .lo-container
+          .pure-g
+            .pure-u-1.pure-u-md-1-2
+              %h2 Community comments
+              .h2-subtitle See what others have said about this solution
+              -@comments.each do |reaction|
+                .comment.pure-g
+                  .pure-u-1.pure-u-md-2-24
+                    -if reaction.emotion == "like"
+                      =reaction_like_icon
+                    -elsif reaction.emotion == "love"
+                      reaction_love_icon
+                    -elsif reaction.emotion == "genius"
+                      =reaction_genius_icon
+                    -else reaction.emotion == "legacy"
+                      =icon 'comment'
+
+                  .pure-u-1.pure-u-md-22-24.content
+                    .when #{time_ago_in_words(reaction.created_at)} ago
```

Comment:
I've heard people argue that `time_ago` should be done client side, not by Rails.

Reply:
Yeah. I don't disagree. I somewhere have a nice library that does both (seeds the text in Rails, live updates it in the client-side). I'll try and work out where I coded that and use it. Thanks for the prompt! :)

Reply:
Sounds brilliant. I'm going to watch out for it.